ProgramParsing/AHS/MajorParser2021_2022.py

```diff
@@ -129,6 +134,9 @@ def load_file(self, file, year):
                         courses = self.__get_courses(information[i])
                         all_courses = all_courses.union(courses)
 
+                        if "Two of" in information[i]:
```

Comment:
This is too hardcoded - If it is one of two of three of - use StringToNumber.py module to convert such that it is `StringToNumber[{use the found regex number}].value`. You can search this up to see how it is used on the other code base